docs/source/en/model_doc/autoformer.md

```diff
@@ -14,30 +14,87 @@ rendered properly in your Markdown viewer.
 
 -->
 
+<div style="float: right;">
+    <div class="flex flex-wrap space-x-1">
+        <img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
+    </div>
+</div>
+
 # Autoformer
 
-<div class="flex flex-wrap space-x-1">
-<img alt="PyTorch" src="https://img.shields.io/badge/PyTorch-DE3412?style=flat&logo=pytorch&logoColor=white">
-</div>
+[Autoformer](https://huggingface.co/papers/2106.13008) is a time series forecasting model that rethinks Transformers with a twist — instead of just using attention, it brings in a **decomposition-based architecture** and a new **Auto-Correlation mechanism** to capture complex patterns in time series data.
+
+What makes Autoformer unique? It doesn't treat trend and seasonal patterns as noise — it learns to **decompose** them, improving both performance and interpretability. Plus, it's super efficient for **long-term forecasting**, avoiding the bottlenecks of traditional Transformer-based approaches.
+
+It’s especially great for scenarios like energy consumption, traffic flow, economic indicators, weather patterns, and disease spread forecasting.
+
+You can explore the model on the [Autoformer Hub Page](https://huggingface.co/elisim/autoformer-energy) or other community-hosted variants.
+
+> [!TIP]
+> Click on the Autoformer model cards in the right sidebar to explore more examples and use cases like weather or energy time series forecasting.
+
+The example below demonstrates how to generate text with [`Pipeline`], [`AutoModel`]
```

Comment:
```suggestion
The example below demonstrates how to make a prediction with [`AutoformerForPrediction`].
```

Reply:
Not resolved yet! I think its better to leave it as how to make a prediction since the docs don't explicitly show how to use it for things like weather and energy